core/services/pipeline/task.eth_call.go

```diff
@@ -45,11 +46,14 @@ func (t *ETHCallTask) Run(ctx context.Context, vars Vars, inputs []Result) (resu
 		contractAddr AddressParam
 		data         BytesParam
 		gas          Uint64Param
+		gasPrice     BigIntParam
```

Comment:
Shouldn't this be a `MaybeBigIntParam` to make the `gasPrice` parameter optional?

Reply:
it is technically optional... (maybe I should rename it) but because big.Ints are pointers. So if it's omitted we just end up with a null pointer.

Reply:
Open to suggestions on how to restructure the param type

Reply:
renamed